nav2_bt_navigator/behavior_trees/navigate_w_recovery_and_replanning_only_if_path_becomes_invalid.xml

```diff
@@ -10,8 +10,11 @@
       <PipelineSequence name="NavigateWithReplanning">
         <RateController hz="1.0">
```

Comment:
Mhm, there's a rate controller here, I think this should be dropped to 2hz so that its every 500ms we can evaluate things.

Reply:
See comment below - make this change only for the new file